runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/config/MqttBindingConfig.java

```diff
@@ -44,8 +45,9 @@ public final class MqttBindingConfig
     public final MqttOptionsConfig options;
     public final List<MqttRouteConfig> routes;
     public final Function<String, String> credentials;
-    public final Map<String, Validator> topics;
+    public final Map<String, ValueValidator> topics;
     public final ToLongFunction<String> resolveId;
+    public final GuardHandler guard;
```

Comment:
Is this field used?

Reply:
yes, in MqttServerFactory class

`sessionAuth = guard.reauthorize(initialId, credentialsMatch);`